src/main/java/org/mule/module/apikit/metadata/internal/amf/MetadataFactory.java

```diff
@@ -16,6 +16,7 @@
 import org.mule.metadata.api.TypeLoader;
 import org.mule.metadata.api.model.MetadataFormat;
 import org.mule.metadata.api.model.MetadataType;
+import org.mule.metadata.api.model.impl.DefaultAnyType;
```

Comment:
Is `org.mule.metadata.api.model.impl` public API? (If it isn't: do we have an alternative?)

Reply:
Good catch, updated!